src/Modules/CrestApps.OrchardCore.AI.Chat/Views/AIChatSessionHeader.cshtml

```diff
@@ -4,13 +4,23 @@
 @model ChatSessionCapsuleViewModel
 
 <div class="card text-bg-theme">
-    <div class="card-body d-flex justify-content-end">
-        <a class="btn btn-secondary me-1" asp-action="Chat" asp-controller="Admin" asp-route-profileId="@Model.Session.ProfileId">
-            <i class="fa-solid fa-plus"></i> @T["New Chat"]
-        </a>
+    <div class="card-body d-flex justify-content-between">
+        <form asp-action="DeleteAll" asp-controller="Admin" asp-route-profileId="@Model.Session.ProfileId" method="post" class="d-inline">
```

Comment:
no need to create a form here. Make it a button only. Adding data-url-af will use global javascript to create post request after the user confirm the action.